shared/src/main/java/com/example/shared/model/Game.java

```diff
@@ -248,60 +244,54 @@ public void sendDeckCount(){
     }
 
     public void cardSelected(String username, int index) {
-        //5 is the index for the face-down-deck
-        for (Player player : playerList) {
-            if (player.getUsername().equals(username)) {
-                if (index != 5) {
-                    player.addCard(trainCardsFaceUp.get(index));
-                    trainCardsFaceUp.set(index, trainCardDeck.drawFromTop());
-                    clientProxy.updateFaceUpCards(trainCardsFaceUp);
-                    clientProxy.sendDeckCount(ticketDeck.getDeckSize(), trainCardDeck.getDeckSize());
-                } else {
-                    TrainCard newCard = trainCardDeck.drawFromTop();
-                    player.addCard(newCard);
-                    clientProxy.receiveFaceDownCard(newCard, username, gameId);
-                    clientProxy.sendDeckCount(ticketDeck.getDeckSize(), trainCardDeck.getDeckSize());
-                }
-            }
+        if (index != 5) {
```

Comment:
*Nit picking*
I think it would be easier to read if you had one statement that said `if(index > 0 && index < 5)` for taking from the pile and `if(index == 5)` for when it's taken from the deck.

Reply:
Another one of my issues. Do as you wish to make the code look better.